apps/language_server/lib/language_server/providers/completion.ex

```diff
@@ -309,7 +309,15 @@ defmodule ElixirLS.LanguageServer.Providers.Completion do
         kind: :keyword,
         detail: "reserved word",
         insert_text:
-          if(String.trim(context.text_after_cursor) == "", do: "do\n  $0\nend", else: "do: "),
+          if String.trim(context.text_after_cursor) == "" do
+            if Keyword.get(options, :snippets_supported, false) do
+              "do\n  $0\nend"
+            else
+              "do"
```

Comment:
It's possible to return `"do\n  "` here, but I'm not sure - for my use-case (Emacs and lsp-mode) pressing return after plain `do` automatically inserts matching `end`, so if elixir-ls returns plain `do` then I can just press return twice to get the desired result (otherwise I will have to type `end` manually).